src/linear_algebra/direct_sum_module.lean

```diff
@@ -65,6 +68,10 @@ theorem to_module.ext (H : ∀ i, ψ.comp (lof R ι M i) = ψ'.comp (lof R ι M
   ψ f = ψ' f :=
 by rw [to_module.unique R ψ, to_module.unique R ψ', funext H]
 
+/--
+The inclusion of the direct sum of a subset of the summands, as a linear map.
+The name seems kind of arcane?
```

Comment:
```suggestion

```

Reply:
I feel like `direct_sum.lset_to_set` doesn't tell me anything. Is there a naming convention I'm missing that allows me to interpret this?

Reply:
Not that I understand! I don't really like the "prefix `l`" naming convention in the first place. In other places we use the much more verbose (but perhaps more readable?) convention that if `FFF` is some set-level function, `FFF.ring_hom` or `FFF.add_monoid_hom` is the corresponding bundled morphism.

Reply:
Hmm. `subset.linear_map`? `subset_inclusion.linear_map`?

Reply:
Let's just leave the name as is for now. There are lots of strangely named `l*` lemmas, and we can think about them all at once someday. :-)